actions/serverless_branch_deploy/action.yml

```diff
@@ -77,14 +77,21 @@ runs:
         env_vars: ${{ inputs.env_vars }}
         base_image: ${{ inputs.base_image }}
 
+    - name: generate short github sha
+      shell: bash
+      id: generate-short-sha
+      run: |
+          SHA="${{ github.sha }}"
+          echo short_sha=${SHA:0:7} >> $GITHUB_OUTPUT
```

Comment:
maybe just write `SHORT_SHA=...  > $GITHUB_ENV` and use it from `{{ env.SHORT_SHA }}`?

the outputs have always seemed more fragile to me to get right.
using an env would also make it available in our scripts if we ever need to debug.